tasks/deploy.js

```diff
@@ -650,6 +659,7 @@ const deployAvalancheDao = async ({
       getEnvVar("DAO_OWNER_ADDR")
     ),
     kycMaxMembers: getOptionalEnvVar("KYC_MAX_MEMBERS", toBN(1000)),
+    kycCanTopUp: getOptionalEnvVar("KYC_CAN_TOP_UP", 0),
```

Comment:
I am wondering if we should leave the top up enabled by default `1`. I think it makes sense to double check that, otherwise it just adds some overhead to update every DAO after the deployment to enable the top up.

Reply:
this makes sense to me. Are there any docs that should be updated to reflect this change in behavior? @adridadou thoughts?